addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -637,7 +637,7 @@ HtmlField.extractProps = ({ attrs, field }) => {
         // can be set elsewhere otherwise.
         wysiwygOptions.allowCommandImage = Boolean(attrs.options.allowCommandImage);
     }
-    if (field.sanitize && field.sanitize_tags) {
+    if (field.sanitize || field.sanitize_tags) {
```

Comment:
@dmo-odoo  I made this change after LNA said the first commit restored the video command where we didn't want it to appear. I noticed in 16.0 it doesn't appear in the full screen composer of the chatter, but it does after the first commit. This fixes it because that particular case has `sanitize === false` but `sanitize_tags === true`. I put a debugger in the python sanitizer and noticed we do indeed go through it, and in the python context, the html field does in fact have `sanitize === true`. So it might be that this code didn't need changing but that in fact it's the value of `field.sanitize` that is wrong in this particular context. I'm lacking information in that regard so I suggest we discuss (huhu) this on Monday.

Reply:
What if `sanitize` is true but not `sanitize_tags` ? I think it is only when `sanitize_tags` is false that we want to get rid of the video.